mypy.ini

```diff
@@ -0,0 +1,53 @@
+[mypy]
+# This is the default. We also run with Python 2.7 in `tox.ini`.
+python_version: 3.5
+
+# TODO: turn on.
+check_untyped_defs = False
+
+no_implicit_optional = True
+
+# TODO: enable once we drop Python 2. Otherwise, we need it because some ignores depend on which
+#  interpreter we use.
+warn_redundant_casts = False
```

Comment:
Does that apply to just this warning, or the top 3? What about the next 3?

Reply:
Top 3. I'll clarify.